flow/embedded_views.h

```diff
@@ -9,23 +9,37 @@
 
 #include "flutter/fml/memory/ref_counted.h"
 #include "third_party/skia/include/core/SkCanvas.h"
+#include "third_party/skia/include/core/SkPath.h"
 #include "third_party/skia/include/core/SkPoint.h"
+#include "third_party/skia/include/core/SkRRect.h"
+#include "third_party/skia/include/core/SkRect.h"
 #include "third_party/skia/include/core/SkSize.h"
 
 namespace flutter {
 
+class FlutterEmbededViewTransformStack;
+class FlutterEmbededViewTransformElement;
+
 class EmbeddedViewParams {
  public:
   SkPoint offsetPixels;
   SkSize sizePoints;
+  std::shared_ptr<FlutterEmbededViewTransformStack> transformStack;
```

Comment:
why a shared pointer?

Reply:
Both `EmbededViewParams` and `ExternalViewEmbedder` holds a reference to this object. I might be wrong but I thought I had to use shared for it?